packages/io/io/index.js

```diff
@@ -1,5 +1,4 @@
 const { makeBlob, convertToBlob } = require('@jscad/io-utils')
-const prepareOutput = require('./prepareOutput')
 
 const amfSerializer = require('@jscad/amf-serializer')
```

Comment:
Seems weird to have them listed here as well.  Can we have only one place where they get registered?

Reply:
Yeah. I almost created a list for serializers as well, but decide to keep this focused on deserializers, and not expand scope.

I’ll add a new issue for the serializers cleanup.